GitUI/GitUIExtensions.cs

```diff
@@ -51,6 +51,12 @@ public static class GitUIExtensions
             [NotNull] string defaultText = "",
             [CanBeNull] Action openWithDiffTool = null)
         {
+            if (file != null && file.ErrorMsg.IsNotNullOrWhitespace())
```

Comment:
I believe we're moving away from these extensions
```suggestion
            if (!string.IsNullOrWhitespace(file?.ErrorMsg))
```

Reply:
ok